### ISSUE TITLE
Make it possible to cut column data instead of word wrapping when using maxColumnWidth

### ISSUE THREAD

**Issue by davsclaus** [freva/ascii-table#12]
Great utility, we started using this with Apache Camel

We print a table with Camel route status, and instead of word wrapping then I would like to make it automatic cut the excess length

```
new Column().header("Name").dataAlign(HorizontalAlign.LEFT).maxColumnWidth(30).with(r -> r.name),
new Column().header("Route ID").dataAlign(HorizontalAlign.LEFT).maxColumnWidth(30).with(r -> r.routeId),
new Column().header("From").dataAlign(HorizontalAlign.LEFT).maxColumnWidth(30).with(r -> r.from),
```

See screenshot for an example of what it does today with word wrap.

**Comment by davsclaus** [comment]
<img width="2130" alt="Screenshot 2022-08-23 at 10 35 28" src="https://user-images.githubusercontent.com/477100/186112318-39e59e7c-d7b0-4b38-8445-86a9666ec890.png">

**Comment by davsclaus** [comment]
For cutting the data (if its string) then you may want to be able to either do a hard cut, or have a suffix text that indicate its cut, such as `xxxxxxx...` eg using 3 dots.

**Comment by freva** [comment]
@davsclaus Could you please take a look at the example in https://github.com/freva/ascii-table/blob/655d546aafa5d8a8c63a57b0ac7bb51c904512fd/README.md#controlling-overflow-behaviour and let me know if this works for you?

**Comment by davsclaus** [comment]
Yes this is perfect, thanks

**Comment by freva** [comment]
Released in 1.4.0
